src/domain/shared/layout/CardsLayout/CardsLayoutScroller.tsx

```diff
@@ -0,0 +1,26 @@
+import { PropsWithChildren } from 'react';
```

Comment:
this is taken out from the CalloutLayout because it's a common thing that we want for the CardsLayout.
Probably we even want to pass a parameter `scrollable` to CardsLayout and include this container within it, but for now I thought this was nicer.